cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,89 @@
 /// <reference types='cypress' />
+import { generateWorker } from '../support/generate';
 
 describe('Web Tables page', () => {
-  it('', () => {
+  const worker = generateWorker();
 
+  function addWorker() {
+    cy.get('#addNewRecordButton').click();
+
+    cy.get('#firstName').type(worker.firstName);
+    cy.get('#lastName').type(worker.lastName);
+    cy.get('#userEmail').type(worker.email);
+    cy.get('#age').type(worker.age);
+    cy.get('#salary').type(worker.salary);
+    cy.get('#department').type(worker.department);
+
+    cy.get('#submit').click();
+
+    return worker;
+  }
+
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it('should add, edit and delete new worker', () => {
+    addWorker();
+
+    cy.get('#searchBox').type(worker.firstName);
+    cy.get('#edit-record-4').click();
```

Comment:
The selector '#edit-record-4' assumes that the new worker is always added at this position. This may not be reliable if the table's state changes between tests or if there are other workers present.

Reply:
But I can't select that element without using '#edit-record-4', there is no text or class there